docker/Dockerfile.docs

```diff
@@ -1,14 +1,15 @@
-FROM python:2.7.14-slim
+FROM python:2.7.15-slim
 
-WORKDIR /app/
-RUN groupadd --gid 10001 app && useradd -g app --uid 10001 --shell /usr/sbin/nologin app
+ARG groupid=10001
+ARG userid=10001
 
-# Install OS-level things
-COPY ./docker/set_up_ubuntu.sh /tmp/
-RUN DEBIAN_FRONTEND=noninteractive /tmp/set_up_ubuntu.sh
+WORKDIR /app/
+RUN groupadd --gid $groupid app && \
+    useradd -g app --uid $userid --shell /usr/sbin/nologin app
 
 # Install graphviz
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y graphviz
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y graphviz make
```

Comment:
This removes a lot of stuff that the docs image doesn't require. Now it builds much faster.